clustered/integration-test/src/test/java/org/ehcache/clustered/TerminatedServerTest.java

```diff
@@ -141,6 +144,9 @@ public static void setProperties() {
     overrideProperty(oldProperties, TCPropertiesConsts.L1_SHUTDOWN_THREADGROUP_GRACETIME, "1000");
     overrideProperty(oldProperties, TCPropertiesConsts.TC_TRANSPORT_HANDSHAKE_TIMEOUT, "1000");
 
+    // Used only by testTerminationFreezesTheClient to be able to fill the inflight queue
+    overrideProperty(oldProperties, TCPropertiesConsts.CLIENT_MAX_PENDING_REQUESTS, "" + CLIENT_MAX_PENDING_REQUESTS);
```

Comment:
Pet peave of mine.  I'd prefer `Integer.toString(CLIENT_MAX_PENDING_REQUESTS)`.

Reply:
Ok

Reply:
hahaha

Sorry Henri, private joke.